bluemira/optimisation/_nlopt/optimiser.py

```diff
@@ -398,11 +398,17 @@ def _process_nlopt_result(opt: nlopt.opt) -> None:
     ----------
     opt:
         The optimiser to check
+    algorithm:
+        The optimisation algorithm
     """
     result = opt.last_optimize_result()
     message = None
+    log_func = bluemira_warn
     if result == nlopt.MAXEVAL_REACHED:
         message = "optimiser succeeded but stopped at the maximum number of evaluations."
+        if algorithm is Algorithm.ISRES:
+            log_func = bluemira_debug
+            message += "\nThis is expected for the {Algorithm.ISRES.name} algorithm."
```

Comment:
Looks like this is missing the f for the f-string?

Reply:
Good catch!